gateway.php

```diff
@@ -29,73 +33,43 @@ public function __construct()
         $this->init_form_fields();
         $this->init_settings();
 
-        // Get variables
-        $this->enabled = $this->get_option('enabled');
-
         // Internal Options
         $this->use_button = ($this->get_option('button') === 'yes');
-        $this->test_mode = ($this->get_option('test_mode') === 'yes');
+        $this->test_mode  = ($this->get_option('test_mode') === 'yes');
         $this->use_wallet = ($this->get_option('wallet') === 'yes');
 
-        // New Webhook
-        $this->use_webhook_api = ($this->get_option('use_webhook_api') === 'yes');
-
         // Enable or Disable financial information in products
         $this->financial_info_active = ($this->get_option('financial_info_active') === 'yes');
 
         // Theme
-        $this->checkout_title = $this->get_option('checkout_title');
-        $this->checkout_logo = $this->get_option('checkout_logo');
-        $this->checkout_theme = $this->get_option('checkout_theme');
+        $this->checkout_title            = $this->get_option('checkout_title');
+        $this->checkout_logo             = $this->get_option('checkout_logo');
+        $this->checkout_theme            = $this->get_option('checkout_theme');
         $this->checkout_background_color = $this->get_option('checkout_background_color');
-        $this->checkout_primary_color = $this->get_option('checkout_primary_color');
+        $this->checkout_primary_color    = $this->get_option('checkout_primary_color');
 
         // DNI fields
         $this->custom_dni = $this->get_option('custom_dni');
-        $this->own_dni = ($this->get_option('own_dni') === 'yes');
+        $this->own_dni    = ($this->get_option('own_dni') === 'yes');
 
         // Reseller ID
         $this->reseller_id = $this->get_option('reseller_id');
 
         // String variables
-        $this->title = $this->get_option('title');
+        $this->title       = $this->get_option('title');
         $this->description = $this->get_option('description');
 
-        $this->api_key = $this->get_option('api-key');
+        $this->api_key      = $this->get_option('api-key');
         $this->access_token = $this->get_option('access-token');
```

Comment:
Muchas de estas opciones ya no se utilizan en la clase. Si queda alguna en uso estaría bueno pasar a utilizar el array de `$this->helper->settings`.